ROBOFLOW ADDED

taniya542 opened the pull request.

taniya542 commented:
![image](https://github.com/foss42/awesome-generative-ai-apis/assets/143064148/56989d5b-2fb9-4703-a258-299d196ddd56)

@ashitaprasad please review the PR and merge.

ashitaprasad commented on README.md:
```diff
@@ -58,6 +58,9 @@ You can start contributing by adding the following:
 | [Voyage AI](https://www.voyageai.com/) | [Link](https://docs.voyageai.com/reference/embeddings-api) | Y | Voyage AI provides API endpoints for embedding and reranking models. |
 | [AI/ML API](https://aimlapi.com/) | [Link](https://docs.aimlapi.com/) | Y |  An API aggregator that provides access to 100+ AI models via a single API. |
+| [OpenAI DALL-E](https://openai.com/index/dall-e-2/) | [Link](https://platform.openai.com/docs/guides/images/image-generation) | Y |  An API aggregator that  provides Creative design, concept art, and advertising from text Prompts. |
+| [Artbreeder](https://www.artbreeder.com/) | [Link](https://www.artbreeder.com/support) | Y | Artbreeder uses generative adversarial networks (GANs) to create and modify images.It's use cases are digital art,character design and genetic algorithms. |
```
You need to add the API documentation link here. Kindly look at other generative ai apis that have been added.

taniya542 replied:
Actually it's documentation is not available.

ashitaprasad commented on README.md:
```diff
@@ -58,6 +58,9 @@ You can start contributing by adding the following:
 | [GooseAI](https://goose.ai/) | [Link](https://goose.ai/docs) | Y | GooseAI provides a fully managed NLP-as-a-Service, offering various GPT-based models with high customization and speed. |
 | [Voyage AI](https://www.voyageai.com/) | [Link](https://docs.voyageai.com/reference/embeddings-api) | Y | Voyage AI provides API endpoints for embedding and reranking models. |
 | [AI/ML API](https://aimlapi.com/) | [Link](https://docs.aimlapi.com/) | Y |  An API aggregator that provides access to 100+ AI models via a single API. |
+| [OpenAI DALL-E](https://openai.com/index/dall-e-2/) | [Link](https://platform.openai.com/docs/guides/images/image-generation) | Y |  An API aggregator that  provides Creative design, concept art, and advertising from text Prompts. |
```
This API has already been added.

taniya542 commented:
@ashitaprasad As OpenAI DELL-E was already added, I have deleted it and the documentation of artbreeder was not available so deleted it too. In place of that, I have added ROBOFLOW. Please review it and suggest any changes if needed.
